pgscatalog_utils/scorefile/effectallele.py

```diff
@@ -0,0 +1,16 @@
+class EffectAllele:
+    _valid_bases = frozenset({"A", "C", "T", "G"})
```

Comment:
Do we need to add the base `-` or do you skip all the non-SNP variants ?

Reply:
@ens-lgil could we point @nebfield to some problematic PGS IDs that would be good for this sort of testing? Basically anything with HLA.

Reply:
I was more thinking about `deletion` variants, but I see the issue with some HLA where the allele are `+` and `-`.
For instance, PGS003917:
```
rsID	chr_name	chr_position	effect_allele	other_allele	effect_weight	is_haplotype	imputation_method	locus_name	variant_description	hm_source	hm_rsID	hm_chr	hm_pos	hm_inferOtherAllele
.	6		31238217	+		-		0.000405	True	HLA*IMP:02	HLA-C	variant_id=HLA-C*0303	liftover		6	31270440
```

Reply:
I've updated and added some [doctests](https://docs.python.org/3/library/doctest.html) with the odd effect alleles 😄 thank you!